velox/common/memory/MemoryAllocator.cpp

```diff
@@ -223,17 +220,9 @@ bool MallocAllocator::allocateNonContiguous(
   std::vector<void*> pages;
   pages.reserve(mix.numSizes);
   for (int32_t i = 0; i < mix.numSizes; ++i) {
-    if (TestValue::enabled()) {
-      // NOTE: if 'injectAllocFailure' is set to true by test callback, then
-      // we break out the loop to trigger a memory allocation failure scenario
-      // which doesn't have all the request memory allocated.
-      bool injectAllocFailure = false;
-      TestValue::adjust(
-          "facebook::velox::memory::MallocAllocator::allocateNonContiguous",
-          &injectAllocFailure);
-      if (injectAllocFailure) {
-        break;
-      }
+    // Trigger allocation failure by breaking out the loop.
+    if (testingInjectFailure(Failure::kAllocate)) {
```

Comment:
This code is a bit hard to read. I wonder if it would be clearer as 

```
if (hasInjectedFailure(Failure::kAllocate))
```

Reply:
I changed to testingHasInjectedFailure